pipeline_steps/multi_node_aio_prepare.groovy

```diff
@@ -91,10 +91,16 @@ def prepare() {
         git url: env.OSA_OPS_REPO, branch: "master"
         sh "git checkout ${env.OSA_OPS_BRANCH}"
       }
-      dir("openstack-ansible-ops/${env.MULTI_NODE_AIO_DIR}") {
+      dir("openstack-ansible-ops/multi-node-aio") {
+        // The multi-node-aio tool is quite modest when it comes to allocating
+        // RAM to VMs -- since we have RAM to spare we double that assigned to
+        // infra nodes.
+        sh """#!/bin/bash
+          sed -i 's/server_vm_ram: 8192/server_vm_ram: 16384/g' playbooks/host_vars/infra*.yml
```

Comment:
This comment conflicts with the commit message which describes the default allocation as insufficient, which is correct?

Reply:
This seems fine however I think it'd better to get some type of an option into the host vars for us to use instead of relying on in place injections.

Reply:
I see your mnaio upstream change @cloudnull, will adapt once that merges.  Thanks!